core/sql/optimizer/RelSample.cpp

```diff
@@ -925,7 +925,6 @@ RelExpr *RelSample::bindNode(BindWA *bindWA)
   RelExpr* myChild = child(0);
   if (myChild->getOperatorType() == REL_SCAN &&
       (static_cast<Scan*>(myChild))->isHbaseTable() &&
-      myChild->selectionPred().entries() == 0 &&
```

Comment:
I would have assumed that we still need this check for a query with a sample clause and a predicate ? Not for update stats, but for some other SELECT statement with SAMPLE. Simple predicates can be pushed down only for regular format rows, for aligned format rows predicate evaluation will be in Trafodion.